lint/rule_name.go

```diff
@@ -29,12 +30,41 @@ const nameSeparator string = "::"
 
 var ruleNameValidator = regexp.MustCompile("^([a-z0-9][a-z0-9-]*(::[a-z0-9][a-z0-9-]*)?)+$")
 
-// NewRuleName creates a RuleName from segments.
+// NewRuleName creates a RuleName from an AIP number and a unique name within
+// that AIP.
+func NewRuleName(aip int, name string) RuleName {
+	// Determine the group.
+	group := ""
+	if aip > 0 && aip < 1000 {
+		group = "core"
+	}
+
+	// Sanity check: If the group does not exist, complain.
+	if group == "" {
+		panic("Invalid AIP; no available group.")
+	}
+
+	// Get the AIP as a four-character string.
+	aipStr := fmt.Sprintf("%d", aip)
+	for len(aipStr) < 4 {
```

Comment:
Consider use `fmt.Sprintf("%04d", api)`

Reply:
Oh, thanks. I checked the docs for that but it was not listed.

Reply:
I saw it before -- so I Google it :)